.github/workflows/test.yml

```diff
@@ -30,7 +30,6 @@ jobs:
         run: |
           npm ci
           npm run build --if-present
-          npm run test
           npm run build-storybook
         env:
          CI: ""
```

Comment:
damn you went brazy on this yml file

Reply:
Haha and that npm run test didn't even work 😭😭 for some reason the jest tests don't exit properly in github actions